lib/evss/disability_compensation_form/service.rb

```diff
@@ -66,8 +66,12 @@ def validate_form526(form_content)
       private
 
       def handle_error(error)
-        if error.is_a?(Common::Client::Errors::ClientError) && error.status != 403 && error.body.is_a?(Hash)
-          save_error_details(error)
+        # Common::Client::Errors::ClientError is raised from Common::Client::Base#request after it rescues
+        # Faraday::ClientError.  EVSS::ErrorMiddleware::EVSSError is raised from EVSS::ErrorMiddleware when
+        # there is a 200-response with an error message in the body
+        if ((error.is_a?(Common::Client::Errors::ClientError) && error.status != 403) ||
+           error.is_a?(EVSS::ErrorMiddleware::EVSSError)) && error.body.is_a?(Hash)
```

Comment:
😱 😱 😱 

🤯🤯🤯

Reply:
Your emojis make a great point and I don't disagree.  The whole point of this PR is "make this behave just like it did in April before EVSS made a change to their responses".  I see that many people have tried to refactor EVSS error handling over the years and each one makes it just a tad more complicated. It seemed like my two choices were "mega refactor, again" or "add another condition".  I'd happily take suggestions for another way.  It's hard though, because lighthouse also uses this code.

Also for context, EVSS' sustainment contract ends this week and I'm not aware of what will happen next.

Reply:
Sorry for the lack of context, I'm in full agreement that, "add another condition" is the way to go here.

I'm pretty sure I once took a crack at trying to undo this. After multiple weeks, I eventually gave up on a huge PR.